lib/Cake/Core/CakeObject.php

```diff
@@ -27,6 +27,7 @@
  *
  * @package       Cake.Core
  */
+#[\AllowDynamicProperties]
```

Comment:
There are many classes that create properties dynamically (Controller class for example) and extends  `CakeObject` class. I added this attribute to mitigate those warnings for now.

Reply:
Yep, I added this too on `Model` class. :+1: